Pod/Classes/VOKCoreDataManager.m

```diff
@@ -256,11 +258,36 @@ - (NSArray *)importArray:(NSArray *)inputArray forClass:(Class)objectClass withC
     contextOrNil = [self safeContext:contextOrNil];
 
     NSArray *existingObjectArray;
+    NSMutableArray *repeatedUniqueKeys = [NSMutableArray array];
 
     if (mapper.uniqueComparisonKey) {
-        NSArray *arrayOfUniqueKeys = [inputArray valueForKeyPath:mapper.foreignUniqueComparisonKey];
+        NSString *foreignUniqueKey = mapper.foreignUniqueComparisonKey;
+        NSArray *arrayOfUniqueKeys = [inputArray valueForKeyPath:foreignUniqueKey];
         //filter out all NSNull's
         arrayOfUniqueKeys = [arrayOfUniqueKeys filteredArrayUsingPredicate:[NSPredicate predicateWithFormat:@"self != nil"]];
+        
+        //Use a counted set to grab any repeated keys and notify the user
+        NSCountedSet *countedKeySet = [[NSCountedSet alloc] initWithArray:arrayOfUniqueKeys];
+        for (id key in countedKeySet) {
```

Comment:
Suggestion: wrap this whole thing (the for loop and conditional below) in a check to see if it's even necessary:

```objc
if (countedKeySet.count < arrayOfUniqueKeys.count)
```

From the NSCountedSet docs:

>  The count method defined by the superclass NSSet has special significance; it returns the number of distinct objects, not the total number of times objects are represented in the set.